presto-clickhouse/src/main/java/com/facebook/presto/plugin/clickhouse/ClickHouseMetadata.java

```diff
@@ -253,6 +253,12 @@ public void renameTable(ConnectorSession session, ConnectorTableHandle table, Sc
         clickHouseClient.renameTable(ClickHouseIdentity.from(session), tableHandle, newTableName);
     }
 
+    public void setTableProperties(ConnectorSession session, ConnectorTableHandle table, Map<String, Object> properties)
+    {
+        ClickHouseTableHandle tableHandle = (ClickHouseTableHandle) table;
+        clickHouseClient.setTableProperties(ClickHouseIdentity.from(session), tableHandle, properties);
+    }
+
```

Comment:
Add @Override annotation here.
Can we add some tests to test the feature support in the connector?

Reply:
Clickhouse server is spawned using docker in presto tests. I skipped the test on purpose as I am not sure how do I test it. cc @yingsu00